hstrat/serialization/_unpack_differentiae.py

```diff
@@ -10,23 +11,25 @@ def unpack_differentiae(
     packed_differentiae: str,
     differentia_bit_width: int,
 ) -> typing.Iterable[int]:
-    """Unpack a sequence of differentiae from into a compact, concatenated
-    base 64 representation.
+    """Unpack a compact, concatenated base 64 representation into
+    a sequence of differentiae.
     """
     _bytes = b64decode(packed_differentiae)
-    bits = BitArray(bytes=_bytes)
 
-    # if padding bits are possible, first byte tells how many are required
-    if differentia_bit_width % 8:
+    if differentia_bit_width in [8, 16, 32, 64]:
+        # bit width is a multiple of 8 -- no padding required
+        dt = np.dtype(f">u{differentia_bit_width // 8}")
```

Comment:
name the floor division result num_bytes and give a comment explaining this numpy dtytoe construtor syntax, maybe link to numpy docs; explain > too

Reply:
Done in 120a3e37